shared/css/common.css

```diff
@@ -591,3 +591,20 @@ html[dir="rtl"] .context-wrapper > .context-preview {
 .clicks-allowed.context-wrapper:hover > .context-info > .context-url {
   text-decoration: underline;
 }
+
+.remote-video-box {
+  width: 100%;
+  height: 100%;
+}
+
+.remote-video-box > .remote-cursor {
+  width: 15px;
+  height: 20px;
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 65534;
+  background-color: red;
```

Comment:
Don't need the top/left/background-color styles. Please keep the rules alphabetical.